scripts/package

```diff
@@ -15,12 +15,19 @@ cp ../bin/pipeline ./pipeline/
 
 TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
+#Tag for dependent images
+PIPELINE_UI_TAG=v1.0.1
+JENKINS_BOOT_TAG=v1.0.0
+JENKINS_SLAVE_TAG=v0.1.0
```

Comment:
Are we hard-coding in the image tags for these images? 

```
rancher/pipeline-ui
rancher/pipeline-jenkins-slave
rancher/jenkins-boot
```

Is there a reason why it's called `jenkins-boot` and not `pipeline-jenkins-boot`?

So when we make a release, the only tag that will change will be for the `rancher/pipeline`?

Reply:
Yes, they are hard-coded and need to be updated if corresponding dependent images are updated.
I will change to ‘pipeline-jenkins-boot’ to keep consistency.
When we make a release, we need to update the dependency tags if they are changed. And only ‘rancher/pipeline’ is affected by the git tag.